src/application-base/ApplicationBase.jsx

```diff
@@ -1,29 +1,30 @@
 /* global TERRA_THEME_CONFIG */
 
 import React, {
-  useRef, useEffect, useMemo, useState,
+  useEffect, useMemo, useState,
 } from 'react';
 import PropTypes from 'prop-types';
-import ThemeProvider from 'terra-theme-provider';
 import { ActiveBreakpointProvider } from 'terra-breakpoints';
 import ThemeContextProvider from 'terra-theme-context/lib/ThemeContextProvider';
-
 import { IntlProvider } from 'react-intl';
+
 import { ApplicationIntlProvider } from '../application-intl';
-import i18nLoader from './private/i18nLoader';
-import { NavigationPromptCheckpoint } from '../navigation-prompt';
+import logger from '../utils/logger';
 
+import i18nLoader from './private/i18nLoader';
 import getBrowserLocale from './private/getBrowserLocale';
 import useTestOverrides from './private/useTestOverrides';
-
 import './private/initializeInert';
 import './baseStyles';
 
 const browserLocale = getBrowserLocale();
 
-// We only need to retrieve the root theme and root theme name once for the life of the application.
-const themeConfig = (typeof (TERRA_THEME_CONFIG) !== 'undefined') ? TERRA_THEME_CONFIG : undefined;
-const rootThemeName = themeConfig?.theme ? themeConfig.theme : 'terra-default-theme';
+// We only need to retrieve the root theme and root theme name once for the life
+// of the application.
+const themeConfig = (typeof (TERRA_THEME_CONFIG) !== 'undefined')
```

Comment:
personal preference and nitpick - i think a single line ternary is easier to understand. i'd just use a regular if statement if we wanted to break this into multiple lines 🤷🏼

Reply:
Can you tell I finally turned on line rules again? I'll see if I can simplify these statements a bit.